### ISSUE TITLE
"Improper" is elitist and arbitrary

### ISSUE THREAD

**Issue by dbooth-boston** [w3c/PWETF#234]
This issue separates out (and supersedes) suggestion 14 from https://github.com/w3c/PWETF/issues/228 .

The title of section 5 is currently "If You've Done Something Improper".  But the word "improper" has connotations of elitism and arbitrariness that would be better to avoid.  Who defines what is 'improper'? Miss Manners?  The Queen of England?  The document should instead focus on preventing harm.

SUGGESTION: Change "Improper" to "Harmful", both in the section title and the body of that section.

**Comment by TzviyaSiegman** [comment]
improper was chosen by @lianqi to make this friendlier to non-English speakers. It was originally "wrong".

**Comment by hober** [comment]
I wonder what @alice or @cynthia think re: "wrong" v. "improper" & if one is preferable to the other for non-native English speakers. They both put a ton of work into the TAG's style guide around simplifying the language we use.

**Comment by wareid** [comment]
@dbooth-boston would it be ok to close this issue?

**Comment by dbooth-boston** [comment]
Unless I missed something, it doesn't look like this issue has been addressed yet, and there's been no feedback at all -- neither for nor against -- on the proposed word change, so I don't understand why we would close it.

Incidentally, another candidate word to consider instead of "improper" would be "offensive".  That word is already used in [the body of the text](https://www.w3.org/Consortium/cepc/cepc-20200716/#mistake).

I suggest asking the group's opinion of these choices (and perhaps other choices, if more are contributed):

- improper
- harmful
- offensive
- wrong

**Comment by cwilso** [comment]
For my part as a response - I am strongly in favor of continuing to use the term "improper".  It is certainly not an arbitrary choice - at a quick definition, Oxford languages defines it as "not in accordance with accepted rules or standards, especially of morality or honesty", which seems pretty close to ideal.  I'm not sure what is meant by "elitist"; it's a pretty common English word, and its usage is not always in reference to Miss Manners.

I would bow to those who don't speak English as their first language to know if it is less accessible - but as Tzviya said, this actually came as a suggestion from @lianqi, whose judgment I would trust.

"Wrong", "offensive" and "harmful" all seem like they would excite defensiveness - "this didn't actually harm anyone", "I wasn't offended by it", and "it's not wrong, you just don't like it."  This document DOES define what is "proper" - the first line is "W3C's Code of Conduct defines expected and unacceptable behaviors".

**Comment by dbooth-boston** [comment]
> This document DOES define what is "proper" - the first line is "W3C's Code of Conduct defines expected and unacceptable behaviors".

Good point.  Perhaps "unacceptable" would be a better choice, since that would directly align with the rest of the document.  In contrast, this section is the _only_ place in the document that uses the term "improper", which implicitly suggests that something else may be intended.

**Comment by dbooth-boston** [comment]
I probably should have further explained.  @cwilso noted:

> Oxford languages defines ["improper"] as "not in accordance with accepted rules or standards, especially of morality or honesty"

The elitist overtone is not in the dictionary.  It's like with the word "thug": the racial overtone that it now carries (to some) is not in the dictionary definition.  But it nonetheless exists.

To understand the elitist overtone of "improper", imagine how this Code might be read by someone who is not so attuned to issues of discrimination, prejudice, marginalization, subtle sexual harassment, etc., and is skeptical of the Code's value -- or even an antagonist.  To such people, the Code may feel like a confusing set of rules that are being imposed on them for dubious purposes by a bunch of highly educated techies who work for the most elite companies in the world.  In essence those rules are telling them that the way they've been living and talking for their whole lives is not good enough -- _they_ are not good enough.  

I think the Code is strengthened if we can avoid the elitist "Miss Manners" overtone, even if that overtone is not perceived by every reader.

@cwilso (and others), what do you think of the word "unacceptable" instead of "improper"?

**Comment by cwilso** [comment]
I think unacceptable is worse than improper.  

The point of this term - of this aspect of the CoC - is to define norms that are expected.  "Unacceptable" has a very hard-stop definition - "not allowable" - that I think will trigger "what I did wasn't that bad"-style pushback.  I also think it only addresses one side of improper behavior - although I'm not a huge fan of the phrase "defines expected and unacceptable behaviors" - because it puts two opposites together with "and" - the point of this section isn't just if you did something unacceptable (i.e. did something in 3.2).  Improper behavior can also be if you failed to have the expected behavior (in 3.1) as well, because that too would be improper, and could cause offense.

As a person of privilege - yes, some of these rules MAY be telling me the way I've been living and talking for my whole life is not good enough.  That is absolutely the point, and I should listen to that, and read through the document until I understand why, and learn how to do my part to work better together with others.  If I fail to understand why or how I am marginalizing those of less privilege, I SHOULD be called on it and learn to do better.  

Whether I am skeptical of the Code's value or not is irrelevant - the Code HAS value in improving our community, and should (and will) be adhered to.  If these rules are confusing, we can elaborate.  But let's be clear that there are no dubious purposes here, and we are not here to claim people are "not good enough" - just that their behavior may need to be better, and give them guidance on how to do so.

**Comment by dbooth-boston** [comment]
> I think unacceptable is worse than improper.

Then we have a much bigger problem, because "acceptable" and "unacceptable" are used throughout the document.

Section 5 is about behaviors that "cause offense", are "wrong" and warrant an apology.  That certainly sounds to me like behavior that is not acceptable, i.e., _unacceptable_ _behavior._

> the point of this section isn't just if you did something unacceptable (i.e. did something in 3.2). 

To my mind, the point of this section _is_ if you did something unacceptable, though unintentionally.

Bear in mind that unacceptable behaviors are not limited to those listed in 3.2.  Section 3.2 clearly says: "This list of unacceptable behaviors does not cover every case" and "Unacceptable behaviors include, but are not limited to . . . ."

> Improper behavior can also be if you failed to have the expected behavior (in 3.1) as well, because that too would be improper, and could cause offense

If the behavior causes offense, is "wrong" and warrants an apology (as section 5 describes), then that certainly sounds to me like unacceptable behavior.  I do not see why we would have a section explaining to people how to apologize for behavior that causes offense and is "wrong", but is somehow still considered "acceptable".
  
    -----
    
Regarding your comments about whether the Code has value or dubious purpose, you don't have to convince me that the Code has value and important purpose.  You are preaching to the choir.  I was just trying to give you insight about how more skeptical or antagonistic readers may feel, because you indicated that you didn't see how "improper" has an elitist "Miss Manners" overtone.

I want this Code to be clear and understandable to the broadest possible audience -- including those who may be skeptical or even antagonistic to it.  Although this Code certainly needs to support marginalized people by providing an enforcement tool, I think its value as an educational tool is even more important.  And for that reason, I think the more we can reach skeptical or even antagonistic readers the better.

I would much rather give skeptical readers insights that help them learn than giving them fodder for their prejudices by using terms that alienate them.

This is also why I've spent so much effort trying to clean up and clarify wording: This Code is an important document.  I don't want to give anyone any excuse to dismiss it as half baked.

**Comment by LJWatson** [comment]
@dbooth-boston wrote:
>this Code certainly needs to support marginalized people by providing an enforcement tool, I think its value as an educational tool is even more important.

If I understand your comment properly, I encourage you to reconsider your words. Placing the importance of educating potential perpetraters over the importance of enabling people from marginalised groups to participate safely in our community, might be construed as somewhat dismissive.

If the CoC educates people about how to behave within our community, it may reduce the number of CoC transgressions, and that in turn might reduce the need for the CoC to be used in defence of/by anyone including people from marginalised groups. Sadly however, it will not eliminate either possibility entirely, so placing education over safety feels like the wrong emphasis to me.

**Comment by dbooth-boston** [comment]
> Placing the importance of educating potential perpetraters over the importance of enabling people from marginalised groups to participate safely in our community

My intent is the exact opposite: it is to _enable_ people from marginalized groups to feel welcome and participate safely by _preventing_ transgressions that would make people feel unwelcome or unsafe.

> placing education over safety

I do _not_ advocate placing education over safety.  

Clearly the enforcement and the educational aspects of the Code are both important, and they function in different ways to help achieve the same goal of a safe and welcoming work environment.  I did not in any way intend to imply that the enforcement aspect is unnecessary or unimportant, and I'm sorry if it somehow appeared as though I did.

I personally view the educational aspect as an ounce of prevention (versus a pound of cure).  But I completely respect your view that the enforcement aspect is more important, and it clearly is in some situations.  I think different people can reasonably weigh these aspects differently, while still working toward the common goal of a safe and welcoming work environment.

**Comment by dbooth-boston** [comment]
Hi Chris.  If I'm understanding you correctly, you are saying that section 5 addresses both unacceptable behavior AND behavior that is _acceptable_ (i.e., not unacceptable) but also not "best" (as described in 3.1).  (As a side note, I think it's important to avoid equating "unacceptable behaviors" with section 3.2, since as 3.2 explicitly emphasizes, there are _many_ unacceptable behaviors that are not listed in 3.2.)

I think I understand your desire to address behavior that is acceptable but not ideal, however:

 - I don't think that's what section 5 currently covers.  Section 5 addresses behavior that is _wrong,_ causes _offense_ and warrants _an apology._  To my mind, behavior that meets those criteria definitely should NOT be considered acceptable, even if it is perhaps only marginally unacceptable.

 - I do not think it would be beneficial to have a section that tells people how to apologize for _acceptable_ behavior.  I think it makes more sense for section 5 to only address _unacceptable_ behavior.

I would rather avoid the "Miss Manners" overtone of "improper", because I do not want anyone to view the Code as frivolous, elitist or arbitrary.  Could you live with changing "improper" to "unacceptable" in this context?

**Comment by cwilso** [comment]
This is deep in the weeds of bikeshedding particular words.  I do not think it is wise to change "improper" to "unacceptable" in this context, no - because then the spec will have three sections:

3.1 Expected Behavior
3.2 Unacceptable Behavior
3.3 Safety versus Comfort

Followed by 

5 If You've Done Something Unacceptable

which appears to refer specifically to 3.2, not applying to behaviors mentioned (or lack thereof) in 3.1 or 3.3.  Using a different term seems like a wise idea to me.  "Wrong" would have been fine by me, but I support the change to "improper" as better for an international audience.

Has anyone else supported a concern about the term "improper" as elitist?

**Comment by dbooth-boston** [comment]
Hi Chris.  If you were not aware of the "Miss Manners" overtone of "improper" then it might seem like bikeshedding.  It certainly is not the _most_ important issue in the document.  I just think the document is strengthened if we can avoid terms that may be triggering to readers who are _different_ from us and may not (yet) be as sympathetic to the PWE goals (but hopefully will eventually become more sympathetic}.

The question of using "unacceptable" here instead of "improper" seems to boil down to this:

 - Is section 5 _really_ intended to cover both unacceptable behavior _and_ behavior that is _acceptable_ -- i.e., not _unacceptable_, whether or not it is specifically itemized in 3.2 -- but not "expected"/best?  

I think not.  And as currently written, it fairly clearly does not.  Specifically, I believe you were mistaken when you asserted:

> [In section 5] we're talking about BOTH unacceptable behaviors (3.2), or failing to have best, proper behavior (3.1)

Perhaps that was the intent at some point, but it is _not_ what section 5 currently says.  I urge you to re-read it.  Note that it refers to behavior that is _wrong_, causes _offense_ and results in a _hurt party_ to the point of warranting an _apology_.  That pretty clearly describes _unacceptable_ behavior.  I cannot imagine this group classifying such behavior as _acceptable_.

Regarding this comment:

> 5 If You've Done Something Unacceptable . . . appears to refer specifically to 3.2, not applying to behaviors mentioned (or lack thereof) in 3.1 or 3.3.

Agreed, but referring to 3.2 seems entirely appropriate here, because: 

 - Section 3.2 makes very clear (twice!) that unacceptable behaviors include more than just the specific examples that are itemized; and 

 - As pointed out above, section 5 is about apologizing for _unacceptable_ behavior, even if it does not currently use that word.

In summary, I see these three benefits in using "unacceptable" instead of "improper" in section 5, and I do not see any downside:

 - It more clearly and accurately classifies the offensive/wrong/hurtful behavior that section 5 references.

 - It is more consistent with the rest of the document -- using the same term that is used throughout.

 - It avoids the elitist "Miss Manners" overtone that "improper" carries.

Can you live with "unacceptable" instead of "improper"?

------------

By the way, there is a capitalization error in Section 5:
s/oh sorry/Oh sorry/

**Comment by cwilso** [comment]
> Perhaps that was the intent at some point, but it is not what section 5 currently says. I urge you to re-read it. 

David, I *have* re-read it.  Repeatedly. 

"As we engage ... we may accidentally cause offense, whether through using unknowingly offensive terminology or through missing social cues. If you realize (or are told) that you have offended someone then take the appropriate steps:"

This focuses on the offense caused - whether through unacceptable behavior like offensive terminology (clearly a reference to Section 3.2) or through "missing social cues" (which is NOT in 3.2, but could easily be covered by such things in 3.1 as not being sensitive to language differences; not thinking from others' points of view; not having empathy... etc, etc.)

You said:
>As pointed out above, section 5 is about apologizing for unacceptable behavior, even if it does not currently use that word.

No, it is not.  It's about acknowledging when you have caused offense, apologizing for the offense (no matter how it was caused), and correcting.

I continue to disagree with you, and I continue to believe that using the term "unacceptable" here is not acceptable; it draws a parallel where there should not be one.  The point of Section 5 is "you've caused offense through some action or inaction" - not "you did something listed (or implied) in 3.2".

The downside of using the term "unacceptable" here is that it implies that only distinct unacceptable behaviors need to be addressed.  Much to the contrary: I believe if you realize (or are told) that, say, you have failed to appreciate cultural differences, or you are dominating discussions, or that you've failed to be sensitive to the challenges of language differences, you absolutely should be following up as per section 5 and admitting you've done something wrong, and correcting for it.  (I would personally have a mild preference for the term "wrong" here - but as was previously stated, the term "improper" was a suggestion by a non-native English speaker in preference to the term "wrong", and I'll happily accept that "wrong" might be too harsh or have other implications to a non-native English speaker. I will note that there is still an instance of "wrong" here - "If you don't understand what you did wrong".)

On top of that, I would have to disagree with your three benefits:

- My point was precisely that Section 5 is not about "your wrong behavior" - it's about "you caused offense, what should you do".   Section 5 is not referencing instances of wrong behavior, it's describing how to respond to having caused offense in any way.
- "Unacceptable" *is* consistently used in the document - it's consistently used to define unacceptable behavior, and it is possible to have caused offense and need to address that offense without having "done something unacceptable".  (see above.). I'll not that "unacceptable" is ONLY used in the document in reference to section 3.2.  (and one reference to unacceptable actions re: Ombuds)
- As above: I disagree entirely that Section 5 is about apologizing for unacceptable behaviors.  It's about addressing (recognizing, apologizing, repairing and correcting) any kind of offense.  This is absolutely the key point of why I believe it is correctly captured today.

What do you think should be done if one fails to live up to the Expected Behavior in Section 3.1, and how do you think that would be fulfilled if this were changed?

**Comment by dbooth-boston** [comment]
Hi Chris.  Thanks for your explanation, and also for your patience and diligence in trying to work this out.  Regarding this comment:

> it is possible to have caused offense and need to address that offense without having "done something unacceptable"

I think it depends on what you mean by "unacceptable".  If you are referring _only_ to behavior that is specifically itemized in 3.2, then I would agree.   But if "unacceptable" refers to _any_ unacceptable behavior -- whether or not it is specifically itemized in 3.2 -- then I would disagree.  And as I explain below, I think it is important to use the second interpretation (which is also the interpretation used in 3.2).

> What do you think should be done if one fails to live up to the Expected Behavior in Section 3.1, and how do you think that would be fulfilled if this were changed?

If the behavior was _acceptable_, then I do not think the CoC should try to address it, even if it falls short of ideal behavior.  If the behavior was _unacceptable_ -- even marginally or unintentionally -- then I think section 5 does a good job of addressing it.

An example of behavior that I would view as _acceptable_ but not ideal would be an observer failing to intervene when witnessing unacceptable behavior.  There are many reasons why someone may not feel able to intervene, and we cannot know each person's internal state or circumstances.  I also think that a person in a position of authority has an _obligation_ to intervene, but that's another matter.  Side note: I wonder if that is stated anywhere in some other document.  Should we state it in this document?

It sounds like you want to be sure that section 5 will cover offenses that are minor or marginally unacceptable, but fall short of being so blatant or serious that they are specifically itemized in 3.2.  That seems very reasonable to me.   On the other hand, I feel strongly that:

  - We should never imply or endorse the idea that "unacceptable behavior" _only_ refers to behavior that is specifically itemized in 3.2.  We would be undercutting our own message if we do.

 - We should be clear that behavior that is _wrong_, causes _offense_, and results in a _hurt party_ is _not_ acceptable, even if it was unintentional or only marginally unacceptable.  

As an attempt to thread this needle, how about if we take the following approach in section 5?

 - Change "improper" to "unacceptable", but

 - In the beginning of the body, explicitly clarify that we are including offenses that are only marginally unacceptable.  For example, the second sentence could be changed to something like:

> If you realize (or are told) that you have offended someone or otherwise done something unacceptable -- even if it was unintentional or only marginally unacceptable -- then take the appropriate steps:

Or, as separate late-breaking idea, how about changing "improper" to "that caused offense"?  So the title of section 5 would read:

> 5. If You've Done Something that Caused Offense".  

And step 1 would read:

> 1. Acknowledge that you've done something that caused offense

Or I guess we could shorten that to:

> 1. Acknowledge that you've caused offense

Would any of these suggestions adequately address your concerns?

**Comment by dbooth-boston** [comment]
Discussed in this call: https://www.w3.org/2023/06/20-pwe-minutes.html
I agreed to drop my proposal.
